test/metabase/api/session_test.clj

```diff
@@ -170,7 +170,7 @@
                                               {"x-forwarded-for" "10.1.2.3"})
               status-code (:status response)]
           (assert (= status-code 401) (str "Unexpected response status code:" status-code))))
-      (dotimes [n 50]
+      (dotimes [n 40]
```

Comment:
what's the purpose of this change?

Reply:
The delay was too long, and it was failing the test (especially with coverage instrumentation). The resulting error string was "wait 9 more seconds" but the regex wanted "wait 1\d more seconds". (Gross, I know.)